resources/js/components/Room/HistoryComponent.vue

```diff
@@ -0,0 +1,420 @@
+<template>
+  <div>
+    <b-overlay  :show="meetingsLoadingError" >
+      <!-- Overlay on loading and errors -->
+      <template #overlay>
+        <div class="text-center my-2">
+          <b-spinner v-if="meetingsLoading" ></b-spinner>
+          <b-button
+            v-else
+            @click="$root.$emit('bv::refresh::table', 'meetings-table')"
+          >
+            <b-icon-arrow-clockwise></b-icon-arrow-clockwise> {{ $t('app.reload') }}
+          </b-button>
+        </div>
+      </template>
+
+      <div class="row">
+        <div class="col-12">
+          <!-- Reload meetings list -->
+          <b-button
+            class="float-right"
+            variant="dark"
+            :disabled="meetingsLoading"
+            @click="$root.$emit('bv::refresh::table', 'meetings-table')"
+            :title="$t('app.reload')"
+            v-b-tooltip.hover
+          >
+            <i class="fas fa-sync"></i>
+          </b-button>
+        </div>
+      </div>
+
+      <!-- List of all meetings -->
+      <div class="row pt-4">
+        <div class="col-12">
+          <b-table
+            fixed
+            hover
+            stacked='lg'
+            :show-empty="!meetingsLoadingError"
+            :busy.sync='meetingsLoading'
+            :fields='meetingsTableFields'
+            :items='fetchMeetings'
+            id='meetings-table'
+            ref='meetings'
+            :current-page='meetingsMeta.current_page'>
+
+            <template v-slot:empty>
+              <i>{{ $t('meetings.nodata') }}</i>
+            </template>
+
+            <template v-slot:table-busy>
+              <div class="text-center my-2" v-if="!meetingsLoadingError">
+                <b-spinner className="align-middle"></b-spinner>
+              </div>
+            </template>
+
+            <template v-slot:cell(start)="data">
+              {{ $date.utc(data.item.start).tz(userTimezone).format('DD.MM.YY HH:mm') }}
```

Comment:
We should use the locale date format with [$d](https://kazupon.github.io/vue-i18n/guide/datetime.html)!